File: src/main/java/com/wolt/domain/deliveryfeecalculator/dto/OrderDataDto.java
package com.wolt.domain.deliveryfeecalculator.dto;

import lombok.Builder;

import java.math.BigInteger;
import java.time.ZonedDateTime;

@Builder
public record OrderDataDto(BigInteger cartValue,
                           BigInteger deliveryDistance,
                           BigInteger numberOfItems,
                           ZonedDateTime orderTime) {
}

File: src/main/java/com/wolt/domain/deliveryfeecalculator/dto/OrderDataDto.kt
package com.wolt.domain.deliveryfeecalculator.dto

import lombok.Builder
import java.math.BigInteger
import java.time.ZonedDateTime

@Builder
@JvmRecord
data class OrderDataDto(val cartValue: BigInteger, val deliveryDistance: BigInteger, val numberOfItems: BigInteger, val orderTime: ZonedDateTime)